content/glossary/hamburger-menu/content.lr

```diff
@@ -0,0 +1,17 @@
+_model: word
+---
+term: hamburger menu
+---
+definition:
+
+It's a icon with three horizontal lines usually on the top-left corner or top-right corner of the screen.
+By clicking or tapping on the icon, it reveals a menu with options or additional pages.
+
+Reference: https://en.wikipedia.org/wiki/Hamburger_button
+
+![hamburger-menu](hamburger-menu.png "Hamburger Menu")
```

Comment:
Hey @luanafaustini, just a small modification:
The image should be moved to `assets/static/images/`
And you will need to modify this markdown path too.